### DISCUSSION TITLE
"anonymous: true" not need?

### DISCUSSION THREAD

**Discussion by cyphos1973** [symfony/symfony#43057] [Q&A] [answered]
Hi
I have this following problem
`The authenticator manager no longer has "anonymous" security. Please remove this option under the "main" firewall.`

but the documentation https://symfony.com/doc/current/security.html#a-authentication-firewalls says :
`anonymous: true`

This is a bug? Or The docs isn't good?

I use symfony 5.3.7
Thanks you
My best regards

**Comment by stof** [comment, marked as answer]
I think this doc page is not updated yet for the new authentication system. See https://symfony.com/doc/current/security/authenticator_manager.html

**Comment by cyphos1973** [comment]
ok thanks you
when the docs will update?

Best regards
